open-metadata-implementation/view-services/README.md

```diff
@@ -0,0 +1,121 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
+  
+# Open Metadata View Services (OMVS)
```

Comment:
Just curious if this an agreed name & abbreviation?  It feels kind of confusing to me to be so general ("Open Metadata View Services") when it is specifically geared to UIs -- when I first saw the name and acronym I was thinking about views in the data sense of the word rather than the MVC-sense of the word...

Not sure what a great alternative would be, though 🤷‍♂

Reply:
I have talked with Mandy about it; nothing more formal. I have been using this abbreviation. It seemed consistent with OMAS and OMRS. Not sure what we would use instead either. I suggest leaving it as is until there is a better alternative